CMakeLists.txt

```diff
@@ -144,16 +145,23 @@ target_compile_options(${PROJECT_NAME}
         >
 )
 
+target_compile_definitions(${PROJECT_NAME}
+    PRIVATE
+        # Enable shared library
+        $<$<BOOL:${BUILD_SHARED_LIBS}>:UNITY_SKIP_DEFAULT_RUNNER>
```

Comment:
What about adding option for `BUILD_SHARED_LIBS`, as described [in cmake docs](https://cmake.org/cmake/help/latest/variable/BUILD_SHARED_LIBS.html)?